### PR TITLE
Increase the maximum content length

### PR DESCRIPTION
**Pull request by rafaucau**
I changed text to mediumText so now pages can contain more content.
#22

### CODE REVIEW

**Comment by clarkwinkelmann** [comment]
I don't think this is the good approach. We need a new migration that converts existing columns, so that everyone gets to the same point after updating. It would be confusing if some existing extension users still have a `TEXT` field in the database.

I would prefer we reach a consensus in the GitHub issue first before approving or denying any pull request.

**Comment by dsevillamartin** [comment]
> I don't think this is the good approach. We need a new migration that converts existing columns

Is that not what this PR does? The last method is `change()` - doesn't that modify an existing column?

**Comment by rafaucau** [comment]
> The last method is change() - doesn't that modify an existing column?

This modifies an existing column without losing any data in it.

**Comment by clarkwinkelmann** [comment]
My bad, I was certain this was a change to the first migration file for some reason :thinking: 

Continue on :+1:

File: src/PageValidator.php [hunk 1]
```diff
@@ -30,7 +30,7 @@ class PageValidator extends AbstractValidator
         ],
         'content' => [
             'required',
-            'max:65535',
+            'max:16777215',
```
Comment (w-4) [inline src/PageValidator.php:33]: I'd even remove this line entirely. No one will write ever write more than 16 million characters.

Comment (rafaucau) [reply src/PageValidator.php:33]: I don't think validation can hurt.
@clarkwinkelmann What do you think?

**Pull request merged**